meta-aos-install-rpi/recipes-initramfs/files/install_script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+SEPARATOR="================================================================="
+
+wait_sync() {
+    sleep 3
```

Comment:
Why sleep is here? I don't think sleep is required before sync at all. Also stand alone function is not required for just sync.